### DISCUSSION TITLE
Health: Progressing for Service in K8s/ArgoCD

### DISCUSSION THREAD

**Discussion by ferdziu7** [argoproj/argo-cd#17638] [Q&A]
Hello, my app is stable but status of health of my service app is "Processing". Maybe problem is that I don't use Ingress proxy instead of external proxy. For example my second service which type is Cluster, status is healthy. It's not critical fail but I just wont to have eveything on green. Maybe there coulde be my mistake in config settings (thats my first project in ArgoCD and k8s so please be kind).

Environments and versions:
Networking k8s: cilium
ArgoCD version: v2.10.4
K8s version: 1.29
self-hosted servers

**Comment by llavaud** [comment]
Hello,
The Service healthcheck is available [here](https://github.com/argoproj/gitops-engine/blob/master/pkg/health/health_service.go#L27-L37), so I guess you are using a service of type LoadBalancer but your ingress controller do not inject the LoadBalancer IP in the `Status.LoadBalancer.Ingress` field.
